### PR TITLE
docs: add Formik examples in storybook 

### PR DESCRIPTION
**Pull request by massao**
# Purpose of PR

Integrates our Form Elements with Formik and creates tests for them
image

I will add documentation in the Website in another PR

### CODE REVIEW

File: packages/components/forms/stories/Formik.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,337 @@
...
+          <Button variant="primary" type="submit">
+            Submit
+          </Button>
+        </Form>
+      )}
+    </Formik>
+  );
+};
+
+export const WithUseFormik = () => {
```
Comment (massao) [inline packages/components/forms/stories/Formik.stories.tsx:228]: I added this example so we can show that it also works with this approach, but they recommend using the `<Formik>` and `<Field>` components

**Comment by denkristoffer** [comment]
My take would to combine these stories into one, do we need to show (and update) this many examples?

**Comment by suevalov** [comment]
I'd probably double up on @denkristoffer suggestions to combine all stories into one with a complex form that represents all our form components.

Otherwise, the PR looks great.

**Review by denkristoffer — APPROVED** [review]
🙏

**Pull request merged**